test/kafka-ssl/mzcompose.py

```diff
@@ -111,4 +119,9 @@ def workflow_default(c: Composition, parser: WorkflowArgumentParser) -> None:
             "materialized",
         ]
     )
-    c.run("testdrive-svc", *args.files)
+    c.run(
+        "testdrive-svc",
+        "--ccsr-password=sekurity",
+        "--ccsr-username=materialize",
```

Comment:
Can you instead add this to the entrypoint, just for consistency with the way we pass `--cert` and such?